projects/control-service/projects/helm_charts/pipelines-control-service/templates/alertmanager.yaml

```diff
@@ -0,0 +1,34 @@
+{{- /*
+  Copyright 2021-2023 VMware, Inc.
+  SPDX-License-Identifier: Apache-2.0
+ */}}
+
+{{- if .Values.alertmanager.enabled }}
+apiVersion: v1
+kind: List
+items:
```

Comment:
Why you are using a list. There's only one resource  `kind: AlertmanagerConfig` 

There's no need to use list. I am guessing this is copy pated from output of kubectl get AlertmanagerConfig which likely return list of configs.